eks/main.tf

```diff
@@ -0,0 +1,69 @@
+resource "aws_s3_bucket" "thanos_metrics_store" {
```

Comment:
Maybe we should create this resource only if we don't pass the attributes of an externally created storage. IMHO this makes sense since we don't want to lose the data when the module is destroyed.

Reply:
We decided that in fact it would be better to have this possibility in order to avoid the possibility of a loss of data when doing a blue/green upgrade of the cluster. This feature will be implemented in a new PR.